grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -124,6 +126,13 @@ class Requisition implements Comparable<Requisition>, Serializable {
     Date dateRejected
     Boolean approvalRequired
 
+    // Temporary until we add the original order associated with this request
+    OrderTypeCode orderTypeCode
+
+    // Fulfillment properties
+    Integer priority
```

Comment:
You could add constraints to match the available DeliveryTypeCodes. In this case, it would be min 1 and max 5